Misc/NEWS.d/next/Library/2018-03-10-20-14-36.bpo-33038.yA6CP5.rst

```diff
@@ -0,0 +1,2 @@
+Using a tempfile.SpooledTemporaryFile object with gzip.GzipFile no longer
```

Comment:
Mention also the case of io.open() with a file descriptor.

Reply:
Hm, I don't think this is a new addition? That wasn't what the bug was aimed at - only for special file objects that (a) have a `name` attribute, but (b) don't set it.

Reply:
This PR fixes a case when *fileobj* has a non-string *name* attribute. `tempfile.SpooledTemporaryFile` is one example. `io.open()` with a file descriptor is other example. Maybe there are other examples in the stdlib. And it can fix also cases with third-party file-like objects.

Reply:
Now I understand - in that case the `name` attribute is an `int`.